swagger.json

```diff
@@ -0,0 +1,193 @@
+{
```

Comment:
Could this get outdated with the code? Maybe we can add some makefile target to get that in sync.

Reply:
Agreed.

Based on @darahayes's comment, it might be good to get this some CI integration too.
I'm not sure if the CI would check the swagger.json is up to date (re-generate and diff, fail if the commited on is different),
of if we shouldn't commit the swagger.json at all.
If we don't have swagger.json in source control, I'm wondering when would it get generated.
Thoughts welcome

Reply:
I think it should be generated at docs gen time, so defer that for when we look into getting generated docs somewhere both for the http and the golang APIs.

No need to add to source control since generation is deterministic and there's docs for it.